src/pages/Registration.js

```diff
@@ -12,33 +14,43 @@ export default function Registration() {
 
   function onSubmit(formData) {
```

Comment:
Ich würde auch hier schauen, ob  ich die Funktion umbenne, vielleicht in registerUser oder so in der Art...

Reply:
aaah ich sehe gerade, dass die funktionsnamen so von formik vorgegeben sind, wenn ich die ändere kommt bei firebase nichts an :(